### PR TITLE
transcribe: choose audio encoding based on file extension

### PR DESCRIPTION
**Pull request by happyalu**
In addition to the wav format, This commit adds support for transcribing files in flac, mp3 and vox formats.

### CODE REVIEW

**Review by renaissanceGeek — APPROVED** [review]
Looks good!

File: cmd/cubic-cli/cmd/transcribe.go [hunk 1]
```diff
@@ -84,7 +84,7 @@ In single file mode:
 In list file mode:
     The FILE_PATH should point to a a file listing multiple audio files
     with the format 'Utterance_ID \t FILE_PATH \n'.
-    Each entry should be on its own line.  
+    Each entry should be on its own line.
     Utterance_IDs may not contain whitespace.
```
Comment (jhollowayj) [inline cmd/cubic-cli/cmd/transcribe.go:89]: Would you mind adding a line to the `longMsg` field about supported file types?  Probably something like this:

```txt
CubicSvr supports the following file formats: [  .wav  .flac  .mp3  .vox  .raw  ].
This command automatically detects the file format based on the file extension.  
```

**Review by jhollowayj — APPROVED** [review]
The code looks good.  I think it needs user facing documentation though.

File: cmd/cubic-cli/cmd/transcribe.go (hunk 1 above)
Comment (happyalu) [reply cmd/cubic-cli/cmd/transcribe.go:89]: done. can you check again if what I added ok OK?

**Review by jhollowayj — APPROVED** [review]

**Pull request merged**